101-traffic-manager-external-endpoint/azuredeploy.json

```diff
@@ -0,0 +1,58 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "dnsname": {
```

Comment:
I think in Portal a feedback is generated if the name exists already. Sadly in our template deploy UI, that's not available :(